dev/com.ibm.websphere.appserver.features/visibility/public/faces-4.0/io.openliberty.faces-4.0.feature

```diff
@@ -37,19 +37,18 @@ IBM-ShortName: faces-4.0
 WLP-AlsoKnownAs: jsf-4.0
 Subsystem-Name: Jakarta Server Faces 4.0
 -features=io.openliberty.jakarta.websocket-2.1, \
-  io.openliberty.servlet.api-6.0, \
+  com.ibm.websphere.appserver.servlet-6.0, \
   io.openliberty.jakarta.xmlBinding-4.0, \
```

Comment:
The pages-3.1 feature enabled the servlet feature and the expression language feature which is why we need to add them directly in the faces-4.0 feature now.